### ISSUE TITLE
Rename 'Remove Wallet' to 'Delete Wallet'

### ISSUE THREAD

**Issue by Jakeeyturner** [IBM-Blockchain/blockchain-vscode-extension#991]
We have `Delete gateway`, `Delete package` and `Delete Wallet`. We should probably make `Remove Wallet` consistent!
## Steps to Reproduce

1.
2.
3.

## Existing issues

- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

## Your Environment

* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

**Comment by EdMoffatt** [comment]
I can see the reason for “remove” as opposed to “delete” on a wallet, because in the case it’s a filesystem wallet you’re not deleting it from wherever it lives on disk, you’re just stopping it from appearing in VS Code (i.e. “removing” it from the view).

Gateways I prefer "deleted" because to re-add them you need to re-build out of a connection profile and an identity - it's not like the gateway still exists somewhere on disk if you remove it from VS Code.

Packages, I’m not sure what the behaviour is there i.e. if you ‘delete’ one in VSCode, does it still exist somewhere on disk or is it gone?

I guess another factor to consider is that changes here will change the command palette commands. I _think_ this is probably ok? Even though we're post-V1 now?

TLDR:
I think either I'm good with how it is at the moment, or I'd rename it to "remove" on smart contract packages too if they still exist on disk after taking the action on them.

Let the debate continue! I'll tag @Jakeeyturner for his thoughts and answers :)

**Comment by cazfletch** [comment]
When you take action on a package they are deleted.

Closing this issue
